### ISSUE TITLE
Focus lost / NVDA stops speaking after list item selected in Excel

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#6453]
After selecting an item in a list in Excel, NVDA stops speaking.  Pressing NVDA+tab at this point reads "List item, read only".  The navigation keys still work to move around the spreadsheet, however NVDA doesn't read.  Pressing alt twice is the quickest way to get focus back.

Note, to create a list:

Press ALT+A for the Data ribbon
Press V for the Data Validation menu
Press V again to open the Data Validation dialog
TAB to "Allow" and choose "List".
TAB to Source, and type:
1,2,3
Press ENTER

**Comment by Qchristensen** [comment]
@michaelDCurran

**Comment by jcsteh** [comment]
P2 because this is very confusing for users, and even though there's a workaround, new users won't realise this.

**Comment by Adriani90** [comment]
This is only reproducible in Excel 2016 / 365 with NVDA 2019.1 Beta 1.
I get following errors in the log while navigating after leaving the dropdown list:

```
IO - speech.speak (11:52:57.032):
Speaking [u'2,3,4  2,3,4']
IO - inputCore.InputManager.executeGesture (11:53:00.335):
Input: kb(laptop):enter
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:00.477):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 996 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:00.486):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1026 and childID 0: [Error -2147467259] Unbekannter Fehler
IO - inputCore.InputManager.executeGesture (11:53:01.681):
Input: kb(laptop):downArrow
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:01.733):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1073 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:01.737):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1074 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:01.747):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1076 and childID 0: [Error -2147467259] Unbekannter Fehler
IO - inputCore.InputManager.executeGesture (11:53:02.676):
Input: kb(laptop):escape
IO - inputCore.InputManager.executeGesture (11:53:03.296):
Input: kb(laptop):rightArrow
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:03.341):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1085 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:03.341):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1086 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:03.355):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1088 and childID 0: [Error -2147467259] Unbekannter Fehler
IO - inputCore.InputManager.executeGesture (11:53:03.581):
Input: kb(laptop):rightArrow
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:03.638):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1097 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:03.641):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1098 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:03.642):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1100 and childID 0: [Error -2147467259] Unbekannter Fehler
IO - inputCore.InputManager.executeGesture (11:53:04.671):
Input: kb(laptop):leftArrow
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:04.721):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1109 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:04.727):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1110 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:04.742):
oleacc.AccessibleObjectFromEvent with window 198698, objectID 1112 and childID 0: [Error -2147467259] Unbekannter Fehler
IO - inputCore.InputManager.executeGesture (11:53:04.897):
Input: kb(laptop):leftArrow
```
cc: @lukaszgo1, @leonardder.

**Comment by Adriani90** [comment]
And this is the developer ing:

```
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (11:53:23.346):
Developer info for navigator object:
name: u''
role: ROLE_LISTITEM
states: 
isFocusable: False
hasFocus: False
Python object: <NVDAObjects.UIA.ListItem object at 0x055818D0>
Python class mro: (<class 'NVDAObjects.UIA.ListItem'>, <class 'NVDAObjects.UIA.UIA'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: ''
location: None
value: exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
appModule: <'excel' (appName u'excel', process ID 2636) at address 5581ff0>
appModule.productName: u'Microsoft Office'
appModule.productVersion: u'16.0.11328.20146'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 984924
windowClassName: u'EXCEL:'
windowControlID: 0
windowStyle: 0
windowThreadID: 0
windowText: u''
displayText: exception: 'NoneType' object is not iterable
UIAElement: <POINTER(IUIAutomationElement) ptr=0xbec9740 at 4df1a80>
UIA automationID: Exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
UIA frameworkID: 
UIA runtimeID: (42, 984924, 4, 3593)
UIA providerDescription: [pid:2636,providerId:0x0 Main(parent link):Unidentified Provider (unmanaged:EXCEL.EXE)]
UIA className: Exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
UIA patterns available: 
DEBUGWARNING - IAccessibleHandler.accessibleObjectFromEvent (11:53:23.431):
oleacc.AccessibleObjectFromEvent with window 264246, objectID 1743 and childID 0: [Error -2147467259] Unbekannter Fehler
DEBUGWARNING - NVDAObjects.UIA.UIA._prefetchUIACacheForPropertyIDs (11:53:23.434):
IUIAutomationElement.buildUpdatedCache failed given IDs of set([30019, 30022, 30086, 30025, 30036, 30070, 30103, 30008, 30009, 30010, 30046, 30079])
DEBUGWARNING - NVDAObjects.UIA.UIA._prefetchUIACacheForPropertyIDs (11:53:23.437):
IUIAutomationElement.buildUpdatedCache failed given IDs of set([30019, 30022, 30086, 30025, 30036, 30070, 30103, 30008, 30009, 30010, 30046, 30079])
DEBUGWARNING - NVDAObjects.UIA.UIA._prefetchUIACacheForPropertyIDs (11:53:23.443):
IUIAutomationElement.buildUpdatedCache failed given IDs of [30090, 30108, 30109, 30110, 30112, 30118, 30127, 30128, 30132, 30136, 30137, 30141, 30149, 30151, 30027, 30028, 30029, 30030, 30031, 30032, 30033, 30034, 30035, 30036, 30037, 30038, 30039, 30040, 30041, 30042, 30043, 30044]
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (11:53:23.450):
Developer info for navigator object:
name: ''
role: ROLE_LISTITEM
states: exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
isFocusable: exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
hasFocus: False
Python object: <NVDAObjects.UIA.ListItem object at 0x055818D0>
Python class mro: (<class 'NVDAObjects.UIA.ListItem'>, <class 'NVDAObjects.UIA.UIA'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <type 'object'>)
description: ''
location: None
value: exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
appModule: <'excel' (appName u'excel', process ID 2636) at address 5581ff0>
appModule.productName: u'Microsoft Office'
appModule.productVersion: u'16.0.11328.20146'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 984924
windowClassName: u'EXCEL:'
windowControlID: 0
windowStyle: 0
windowThreadID: 0
windowText: u''
displayText: exception: 'NoneType' object is not iterable
UIAElement: <POINTER(IUIAutomationElement) ptr=0xbec9740 at 4df1a80>
UIA automationID: Exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
UIA frameworkID: 
UIA runtimeID: (42, 984924, 4, 3593)
UIA providerDescription: [pid:2636,providerId:0x0 Main(parent link):Unidentified Provider (unmanaged:EXCEL.EXE)]
UIA className: Exception: (-2147220991, 'Ein Ereignis konnte keinen Abonnenten aufrufen.', (None, None, None, 0, None))
UIA patterns available: 
```

**Comment by Adriani90** [comment]
This seems to happen only when UIA in Microsoft Excel is disabled in NVDA
s advanced settings. With UIA enabled the issue does not occur.
